addons/l10n_fr_pos_cert/static/src/js/Chrome.js

```diff
@@ -19,7 +19,7 @@ odoo.define('l10n_fr_pos_cert.Chrome', function (require) {
                             const info = await this.env.pos.getClosePosInfo();
                             this.showPopup('ClosePosPopup', { info: info });
                         } catch (e) {
-                            if (identifyError(e) instanceof ConnectionLostError||ConnectionAbortedError) {
+                            if ([ConnectionLostError, ConnectionAbortedError].includes(identifyError(e).constructor)) {
```

Comment:
```suggestion
                            if (isConnectionError(e)) {
```
import `isConnectionError` from `point_of_sale/**/utils.js file`.